docker/Dockerfile

```diff
@@ -4,6 +4,10 @@ FROM python:3.8.13-slim
 RUN apt-get update && apt-get install apt-transport-https ca-certificates gnupg curl ssh -y
 RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg  add - && apt-get update -y && apt-get install google-cloud-cli -y
 
+# Install docker to enable docker in docker functionality
+RUN curl -fsSL https://get.docker.com -o get-docker.sh
```

Comment:
I would suggest to use the apt-get packages to install this option instead of curling into the unknown :-). Side effect is that is signed